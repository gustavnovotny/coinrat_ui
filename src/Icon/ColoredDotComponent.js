// @flow
import React from "react"

type Props = {
 color: string
}

const ColoredDotComponent = ({color}: Props) => {
 return <span className="pt-icon pt-icon-dot" style={{color: color}}/>
}

export default ColoredDotComponent
