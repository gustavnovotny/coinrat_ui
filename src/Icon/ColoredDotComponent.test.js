// @flow
import React from 'react'
import renderer from 'react-test-renderer'
import ColoredDotComponent from "./ColoredDotComponent"

it('renders correctly', () => {
  const tree = renderer
    .create(<ColoredDotComponent color="red"/>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
