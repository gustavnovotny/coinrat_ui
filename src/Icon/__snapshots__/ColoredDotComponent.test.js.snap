// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  className="pt-icon pt-icon-dot"
  style={
    Object {
      "color": "red",
    }
  }
/>
`;
